lib/bookmarks.js

```diff
@@ -0,0 +1,39 @@
+import bookmarkModel from '../models';
```

Comment:
What's the purpose of this file?
What is it solving for?

Reply:
it was supposed to be Model my bad, will change that immediately ... I brought the model into the lib file to query the database. to avoid doing that in the middleware   @Chidi3b3r3

Reply:
I have reverted to use 'Model'